pages/linux/iwctl.md

```diff
@@ -0,0 +1,27 @@
+# iwctl
+
+> A command line tool for controlling the iwd network supplicant.
```

Comment:
I wonder if the link to the official wiki page is useful.
```suggestion
> A command line tool for controlling the iwd network supplicant.
> More information: <https://iwd.wiki.kernel.org/gettingstarted>.
```

Reply:
I did think about that, but it doesn't look that useful. 🤔

Reply:
at least it's better to know where start reading.

Reply:
Fair point. 👍🏻

Reply:
Plase @D34DPlayer: could you add the link in both pages so we can merge this? Thanks!